### PR TITLE
Add jQuery-linkify v2.1.4 w/ npm auto-update & homepage update

### PR DESCRIPTION
**Pull request by hare1039**
Pull request for issue: #10606
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/SoapBox/linkifyjs
 * Official website (optional, not the repository): http://soapbox.github.io/linkifyjs/
 * NPM package url (optional):  https://www.npmjs.com/package/linkifyjs
 * License and its reference:
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 27
   - Count of stars: 328
   - Count of forks: 74
 * NPM download stats (optional):
   - Downloads in the last day: 812
   - Downloads in the last week: 5340
   - Downloads in the last month: 22542

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [ ] This lib was not found on cdnjs repo
 * [ ] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by hare1039** [comment]
@cdnjs/intern3 Please help me review on this PR. Thanks.

According to #10606, the library is broken up into many js files which I include them in the src but not `simple-html-tokenizer.js`.
Note these files cannot minify using `minify.sh`

File: ajax/libs/jQuery-linkify/2.1.4/linkify.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import {inherits} from './linkify/utils/class';
+import * as options from './linkify/utils/options';
+import * as scanner from './linkify/core/scanner';
+import * as parser from './linkify/core/parser';
```
Comment (sufuf3) [inline ajax/libs/jQuery-linkify/2.1.4/linkify.js]: Does this library not need the `linkify` folder?

Comment (hare1039) [inline ajax/libs/jQuery-linkify/2.1.4/linkify.js]: @sufuf3 Sorry that I missed that part. Please review this PR again. Thanks.

Note these new files cannot minify neither.

File: ajax/libs/jQuery-linkify/package.json [hunk 2]
```diff
@@ -13,16 +13,17 @@
     "linkify"
   ],
   "homepage": "https://github.com/SoapBox/linkifyjs/",
```
Comment (clairetsai818) [inline ajax/libs/jQuery-linkify/package.json]: [This website](http://soapbox.github.io/linkifyjs/) might be a better choice to be used as the homepage info.

Comment (hare1039) [inline ajax/libs/jQuery-linkify/package.json]: @clairetsai818 I fix this one. Please review this PR again. Thanks.

**Comment by hare1039** [comment]
@pvnr0082t Please help me review on this PR. Thanks.

**Comment by hare1039** [comment]
@pvnr0082t Ok I fix the auto-update and the commit message. Please review this PR again. Thanks.

File: ajax/libs/jQuery-linkify/package.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "jQuery-linkify",
   "description": "Intelligent link recognition, made easy",
-  "filename": "jquery.linkify.min.js",
+  "filename": "linkify.js",
```
Comment (pvnr0082t) [inline ajax/libs/jQuery-linkify/package.json]: Why don't you directly correct it? Why did you change it into `"linkify.min.js"` in 2c45b1a3c1bf967303030e53c8e3608e5ac9269f?

**Comment by hare1039** [comment]
@pvnr0082t I fix the commit messages. Please review this PR again. Thanks.

**Comment by hare1039** [comment]
@pvnr0082t I changed the commit message as you suggest. Please review this PR again. Thanks.

**Comment by hare1039** [comment]
ping

**Comment by PeterDaveHello** [comment]
Any updates here? Thanks!

**Comment by hare1039** [comment]
@kennynaoh @x09326 Please help me review no this PR. Thanks.

**Comment by hare1039** [comment]
@pvnr0082t I try to improve the commit massage. Please review this PR again. Thanks.

**Comment by hare1039** [comment]
@pvnr0082t I place the `close` at the end. Do I need to do that for all PR currently I am sending?
And please review this PR again. Thanks.

**Comment by hare1039** [comment]
@pvnr0082t I fix the last commit message. Please review this PR again. Thanks.

File: ajax/libs/jQuery-linkify/package.json [hunk 4]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "jQuery-linkify",
```
Comment (PeterDaveHello) [inline ajax/libs/jQuery-linkify/package.json]: It's `jQuery-linkify` on CDNJS, update all your commit message to this one.

**Comment by hare1039** [comment]
@pvnr0082t I fix the commit message. Please review this PR again. Thanks.

**Comment by PeterDaveHello** [comment]
Don't forget to mention the filename change next time.
